torchreid/optim/lr_finder.py

```diff
@@ -11,6 +10,50 @@
 import datetime
 
 from torchreid.utils import get_model_attr
+from torchreid.optim import SAM
+
+
+class WrappedLRFinder(LRFinder):
```

Comment:
I suggest using a fixed version of torch-lr-finder in requirements since now you rely on internal details of implementation.

Reply:
Agree